phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexToolIT.java

```diff
@@ -697,20 +691,19 @@ private static void verifyMapper(Job job, boolean directApi, boolean useSnapshot
         }
         try (Connection conn =
                      DriverManager.getConnection(getUrl(), props)) {
-            PTable dataTable = PhoenixRuntime.getTable(conn, SchemaUtil.getTableName(schemaName, dataTableName));
-            PTable indexTable = PhoenixRuntime.getTable(conn, SchemaUtil.getTableName(schemaName, indexTableName));
+            PTable
+                    dataTable =
```

Comment:
nit: can we fix the line breaks here?

Reply:
I liked the old way of these lines even if they were a bit longer, were more readable.